src/Blazored.Toast.TestExtensions/InMemoryToastService.cs

```diff
@@ -0,0 +1,118 @@
+﻿using Blazored.Toast.Services;
+using Microsoft.AspNetCore.Components;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Blazored.Toast.TestExtensions
+{
+    public class InMemoryToastService : IToastService
+    {
+        public IList<InMemoryToast> Toasts = new List<InMemoryToast>();
```

Comment:
I suggest not exposing a mutable list to the users. They dont need to add or change the toasts list. 

```suggestion
        private readonly List<InMemoryToast> toasts =  new();
        public IReadOnlyList<InMemoryToast> Toasts => toasts;
```